marketing-guidelines.md

```diff
@@ -0,0 +1,99 @@
+  * Emphasize that OTel would not be possible without collaboration from many
+    contributors who happen to work for competing vendors and providers
+  * Cite names of the other contributors and vendors involved with OTel efforts
+  * Emphasize our common goals as a community to improve end user/developer
+    experiences and empower them
+* Don’ts:
+  * Imply that a single provider is responsible for OTel itself, and/or one of
+    its various component parts
+  * Diminish the contributions of another organization or of another individual
+
+### II: It’s not a competition
+
+* Do’s:
+  * Emphasize that all contributions are valuable, and that they come in many
+    shapes and sizes, including:
+  * Contributions to the core project code or to language- or
+    framework-specific SDKs
+  * Creating and sharing educational resources (videos, workshops, articles),
+    or shared resources that can be used for educational purposes (e.g. a
+    sample app using specific language/framework)
+  * Community-building activities such as organizing an event or meetup group
+  * Publicly recognize and thank other organizations for their contributions to
+    OTel
+* Don’ts:
+  * Directly compare the volume or value of different contributors to OTel
+    (E.g., via [CNCF devstats](https://devstats.cncf.io/))
+  * Imply that infrequent or minor contributors to OTel are necessarily
+    second-class citizens, and/or that their own OTel compatibility should be
+    questioned as a result (in fact, there’s no reason that any provider needs
+    to contribute to OTel in order to support it)
+
+### III: Promote awareness of OTel interoperability and modularization
+
+* Do’s:
+  * “Shout from the rooftops” about OTel compatibility – the more that
+    end-users understand what they can do with OTel data, the better
+  * Emphasize the vendor-neutrality and portability of any OTel integration
+* Don’ts:
+  * Imply that an end-user isn’t “Using OTel” unless they’re using some
+    specific set of components within OTel (OTel is a “wide” project with many
+    decoupled components)
+  * Publicly denigrate the OTel support of another provider, particularly
+    without objective evidence
+
+## Concerns and Consequences
+
+Inevitably there will be instances where vendors (or at least their Marketing
+departments) run afoul of these guidelines. To date, this hasn’t happened
+frequently, so we don’t want to create an over-complicated process to handle
+concerns.
+
+Here is how we handle such circumstances:
+
+1. Whomever notices the relevant public (marketing) content should write an
+   email to cncf-opentelemetry-governance@lists.cncf.io and include an
+   explanation of why the content is problematic, ideally referencing the
+   [relevant guidelines above](#goals-and-guidelines)
+1. The OTel Governance Committee (GC) will discuss the case during its next
+   (weekly) meeting, or asynchronously via email if possible
```

Comment:
please note here, that some of such concerns may be discussed off the record - with recording turned off.

Reply:
but we guarantee the response e-mail

Reply:
fwiw, I feel "asynchronously via email" covers this well, I don't think we need to declare anything about "off the record" conversations.

Reply:
Do we need to provide a venue for anonymous reports? I don’t feel too strong as person can anonimize herself by using a fake email.

Reply:
I added a "within two weeks" SLA for an email response (much like the recent TC SLA around code donations)

Reply:
Re "anonymous reports": I agree that it's easy enough to create an anonymous email address. I guess we could construct a google form, but TBH I'd be more worried that it falls out of date and sends to `/dev/null` with nobody testing periodically and/or noticing. :-/